apps/nextjs-website/src/lib/api.ts

```diff
@@ -128,3 +131,14 @@ export async function getTutorialLists(productSlug?: string) {
     ) || null;
   return manageUndefinedAndAddProduct(props);
 }
+
+export async function getWebinars(): Promise<readonly Webinar[]> {
+  // TODO: Remove this condition when the webinar content is ready
+  return isProduction ? [] : [...webinars];
+}
+
+export async function getNextWebinars(): Promise<readonly Webinar[]> {
+  return (await getWebinars()).filter(
+    ({ startDateTime }) => startDateTime && startDateTime > new Date()
```

Comment:
```suggestion
    ({ startDateTime }) => startDateTime && startDateTime.getTime() > new Date().getTime()
```
in this case you are comparing numbers, starting from epoch, which is safer